Azure/sonic-buildimage
# [syncd-rpc] Fix PTF dependency in syncd-rpc image

Signed-off-by: Danny Allen <daall@microsoft.com>



**- What I did**
Added PTF as a dependency to the RPC image to ensure that it's installed for the ptf_nn_agent to use.

**- How I did it**
Followed the steps from this PR: https://github.com/Azure/sonic-buildimage/pull/4279

**- How to verify it**
Load the built rpc image onto a DUT and verify that:
1. ptf is installed
2. `scapy` can be run
3. `ptf_nn_agent` is running as expected

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
